plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -179,8 +192,7 @@ export async function createRouter(
   const logger = parentLogger.child({ plugin: 'scaffolder' });
 
   const identity: IdentityApi =
-    options.identity || buildDefaultIdentityClient({ logger });
-
+    options.identity || buildDefaultIdentityClient({ options });
```

Comment:
I think this should this be like this?

```suggestion
    options.identity || buildDefaultIdentityClient(options);
```